### ISSUE TITLE
460 - Unable to edit Partner reporting requirements after amendment

### ISSUE THREAD

**Issue by KKaselkar** [unicef/etools-issues#1051]
Browser: Chrome
Environment: Demo
Workspace: Philippines

1. Create or select a Signed PD/SSFA
2. Add Amendment

Notice that most fields become editable, but not the fields in the Partner Reporting Requirements card.

Test Case says that user should be able to edit the Partner Reporting Requirements after amending the PD/SSFA

**Comment by KKaselkar** [comment]
![image](https://user-images.githubusercontent.com/32198614/34541833-d6d1d28a-f0a7-11e7-9e64-c417aadb125c.png)

**Comment by adi130987** [comment]
There is a new requirement for this here: #1049

**Comment by robertavram** [comment]
A misunderstanding here with the test cases.
The system was designed with un-changeable reporting requirements until we clarified how to re-generate reports and what the validation is on re-shaping reporting periods. If one chooses to re-shape the reporting period for a report that was already submitted that would be problematic...
I believe we reached an agreement on the validation which is documented in the CR doc.
